packages/esm-form-entry-app/src/app/form-schema/form-schema.service.ts

```diff
@@ -6,18 +6,31 @@ import { FormResourceService } from '../openmrs-api/form-resource.service';
 import { FormSchemaCompiler } from '@ampath-kenya/ngx-formentry';
 import { LocalStorageService } from '../local-storage/local-storage.service';
 import { FormSchema, Questions } from '../types';
+import * as ngxTranslate from '@ngx-translate/core';
 
 @Injectable()
 export class FormSchemaService {
   constructor(
     private formsResourceService: FormResourceService,
     private localStorage: LocalStorageService,
     private formSchemaCompiler: FormSchemaCompiler,
+    private translate: ngxTranslate.TranslateService,
   ) {}
 
   public getFormSchemaByUuid(formUuid: string, cached: boolean = true): Observable<FormSchema> {
     const cachedCompiledSchema = this.getCachedCompiledSchemaByUuid(formUuid);
+    const currentLang = this.localStorage.getItem('i18nextLng') || 'en';
```

Comment:
This seems a bit hacky to me, especially when we can just use `window.i18next.language`...